### DISCUSSION TITLE
Can we patch our own versions of Photoshop?

### DISCUSSION THREAD

**Discussion by Adidiii** [LinSoftWin/Photoshop-CC2022-Linux#117] [Q&A]
I've seen multiple people say we can't use our own version of Photoshop because the author has modded his to work on Linux. To that I say... why can't we just patch ours? Please release the steps needed to patch Photoshop.

**Comment by BrycensRanch** [comment]
its a trade secret. SMH

**Comment by Adidiii** [comment]
I realize it's probably GenP + a dual boot. Why can't that information be released officially?
